reals.tex

```diff
@@ -177,7 +177,8 @@ \section{Dedekind reals}
   : \Q \to \Omega$ which is:
   %
   \begin{enumerate}
-  \item \emph{inhabited:} $\exis{q : \Q} L(q)$ and $\exis{r : \Q} U(r)$,
+  \item \label{defn:dedekind-reals-inhabited}
+    \emph{inhabited (i.e. bounded):} $\exis{q : \Q} L(q)$ and $\exis{r : \Q} U(r)$,
```

Comment:
after each "i.e." should come a comma

Reply:
> after each "i.e." should come a comma

I've pushed up a commit to add a comma after "i.e."

Reply:
Or a backslash.  Or replace the space by a tilde.